### PR TITLE
Update java-agent-configuration-config-file.mdx

### PR DESCRIPTION
**Pull request by meda123**
Adding a new environment variable - do not publish until confirmation from Meda.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx [hunk 1]
```diff
@@ -1362,6 +1362,38 @@ For agent versions 4.10.0 or higher the following environment variables are avai
...
+          </th>
+
+          <td>
+             (none)
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    
+  Setting NEW_RELIC_METADATA_SERVICE_VERSION will create the tag, tag.service.version on error data.
```
Comment (obarney) [inline src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx:1394]: We need to either explain here what the service.version is and what value setting it gives the user or link to a place that does.

Comment (obarney) [reply src/content/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file.mdx:1394]: It should say event data not just error data.

**Comment by rhetoric101** [comment]
@meda123 I am converting this to a draft. Let us know when you want us to review it.

**Review by meda123 — COMMENTED** [review]
Updated based on Oscar's comments.

**Review by obarney — APPROVED** [review]
looks good to me

**Comment by akristen** [comment]
Confirmed with @meda123 that this PR was handled already. Am closing + archiving from our Draft board!

**Pull request closed without merging**